pkg/controllers/monitoring-stack/components.go

```diff
@@ -262,7 +293,7 @@ func stackComponentPatchers(ms *stack.MonitoringStack, instanceSelectorKey strin
 
 func newGrafanaDataSource(ms *stack.MonitoringStack) *grafanav1alpha1.GrafanaDataSource {
 	datasourceName := fmt.Sprintf("ms-%s-%s", ms.GetNamespace(), ms.GetName())
-	prometheusURL := fmt.Sprintf("prometheus-operated.%s:9090", ms.GetNamespace())
+	prometheusURL := fmt.Sprintf("%s-prometheus.%s:9090", ms.GetName(), ms.GetNamespace())
```

Comment:
isn't `ms.Name` and `ms.GetName()` the same?
Same with `GetNamespace()` ?

Reply:
Yes, they're the same. Let's stick with the property for consistency.